va/va.h

```diff
@@ -1916,6 +2016,82 @@ typedef struct _VAContextParameterUpdateBuffer
     uint32_t reserved[VA_PADDING_MEDIUM];
 } VAContextParameterUpdateBuffer;
 
+/** \brief AES CTR fullsample encryption type */
+#define VA_ENCRYPTION_TYPE_FULLSAMPLE_CTR       0x00000001
+/** \brief AES CBC fullsample encryption type */
+#define VA_ENCRYPTION_TYPE_FULLSAMPLE_CBC       0x00000002
+/** \brief AES CTR fullsample encryption type */
+#define VA_ENCRYPTION_TYPE_SUBSAMPLE_CTR        0x00000004
+/** \brief AES CBC fullsample encryption type */
+#define VA_ENCRYPTION_TYPE_SUBSAMPLE_CBC        0x00000008
+
+/** \brief structure for encrypted segment info. */
+typedef struct _VAEncryptionSegmentInfo {
+  /** \brief  The offset relative to the start of the bitstream input in
+   *  bytes of the start of the segment */
+  uint32_t segment_start_offset;
+  /** \brief  The length of the segments in bytes */
+  uint32_t segment_length;
+  /** \brief  The length in bytes of the remainder of an incomplete block
+   *  from a previous segment*/
+  uint32_t partial_aes_block_size;
+  /** \brief  The length in bytes of the initial clear data */
+  uint32_t init_byte_length;
+  /** \brief  This will be AES counter for secure decode and secure encode
+   *  when numSegments equals 1, valid size is specified by
+   * \c key_blob_size */
+  uint8_t aes_cbc_iv_or_ctr[64];
+  /** \brief Reserved bytes for future use, must be zero */
+  uint32_t va_reserved[VA_PADDING_MEDIUM];
+} VAEncryptionSegmentInfo;
+
+/** \brief Encryption parameters buffer for VAEncryptionParameterBufferType */
+typedef struct _VAEncryptionParameters {
+  /** \brief Encryption type, refer to \c VA_ENCRYPTION_TYPE_FULLSAMPLE_CTR,
+   * \c VA_ENCRYPTION_TYPE_FULLSAMPLE_CBC, \c VA_ENCRYPTION_TYPE_SUBSAMPLE_CTR,
+   * or \c VA_ENCRYPTION_TYPE_SUBSAMPLE_CBC
+  uint32_t encryption_type;
```

Comment:
The above comment isn't closed properly and encryption_type is now commented out.

Reply:
Fixed, thank you.